Dockerfile

```diff
@@ -103,14 +110,15 @@ LABEL name="Vault Secrets Operator" \
       version=$PRODUCT_VERSION \
       release=$PRODUCT_VERSION \
       revision=$PRODUCT_REVISION \
+      org.opencontainers.image.licenses="BUSL-1.1" \
       summary="The Vault Secrets Operator (VSO) allows Pods to consume Vault secrets natively from Kubernetes Secrets." \
       description="The Vault Secrets Operator (VSO) allows Pods to consume Vault secrets natively from Kubernetes Secrets."
 
 WORKDIR /
 
 COPY dist/$TARGETOS/$TARGETARCH/$BIN_NAME /$BIN_NAME
 COPY dist/$TARGETOS/$TARGETARCH/scripts /scripts
-COPY LICENSE /licenses/copyright.txt
```

Comment:
I think the RedHat certification HasLicense check needs the license to also be present in `/licenses/`? (See https://github.com/hashicorp/vault/pull/28216)

Reply:
Fixed in e2f0b354cca04b9863b2707fc8fbcbeeff399c83